src/lib/facebook.ts

```diff
@@ -128,7 +128,7 @@ export async function getFacebookPosts(): Promise<
   return (
     posts
       // Ensure the posts are not empty (e.g. timeline events like being born)
-      .filter((post) => post.message)
+      .filter((post) => Boolean(post.message))
```

Comment:
Imo lepiej jest używać `!!` [niż Boolean](https://airbnb.io/javascript/#coercion--booleans)

W tym przypadku jeśli chcesz być bardziej explicit to mógłbyś nawet tak zrobić:

```suggestion
      .filter((post) => post.message != null && post.message.length > 0)
```